python/ray/_private/workers/default_worker.py

```diff
@@ -273,10 +274,13 @@
     worker = ray._private.worker.global_worker
 
     # Setup log file.
-    out_file, err_file = node.get_log_file_handles(
-        get_worker_log_file_name(args.worker_type)
+    out_file, err_file = node.get_log_file_handles_with_rotation(
+        get_worker_log_file_name(args.worker_type),
+        rotation_max_size=args.logging_rotate_bytes,
+        rotation_file_num=args.logging_rotate_backup_count,
     )
-    configure_log_file(out_file, err_file)
+    sys.stdout = out_file
```

Comment:
This won't work if user's actor/task calls c++ code and in c++ there is `std::cout << "xxxx"`

Reply:
redo in https://github.com/ray-project/ray/pull/49548